### PR TITLE
Validate JIMMY

### PR DESCRIPTION
**Pull request by smallinvestor**
# Validate [JIMMY](https://solscan.io/token/94XduSfSnyas7jAEFSJSXiCi1xQ4mENWcii1aCvjVuqu)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/jimmycoinsol/status/1659353316864802817
- Coingecko/ CMC URL (If available): https://coinmarketcap.com/dexscan/solana/3bkCHXmjjTvp23JRgokb1jbvoBTcpyGDKroyKmzbU4eV/

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by Pandybary** [comment]
I proved infomations and i can say that everything is correct.

Mint address: 94XduSfSnyas7jAEFSJSXiCi1xQ4mENWcii1aCvjVuqu
Name: JIMMY
Symbol: JIMMY
Token Decimals: 9
Token icon: https://www.jimmycoinsol.vip/static/media/logo.a7fcc2c98544dcad25a5.png
On-chain metadata matches
No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -732,3 +732,4 @@ ANON,$ANON,4PEMPJXiYd4THpNuaFTf2zB8jKYJJPoEN86vdvKnjv9R,7,https://arweave.net/hS
 Food Token,FOOD,EcK2evV2cDECVsmvY2FxU51eu3fp4w48zrZxuA92AAAN,9,https://arweave.net/mq0Euf7EQja8Hr1s7TvhfzLi5GeDWMe3ryvnNJItP3c,true
 PRNT,PRNT,4TUNzcgp2fPD48fcW4seRjyqyDZMrPj4ZubnXFEsKeYk,7,https://shdw-drive.genesysgo.net/EYH4kEfffWwRnF63G6Sx68sHdhkue4trzuSeUji9UJSe/prnt.jpg,true
 TURBO on Solana,$TURBO,42KmnheUHt3UaQFoAKnipLgBJ1Zm427heJdQcHGPtnjc,7,https://arweave.net/FVrzsx6r7mwCakD64Axt59qAB_g-nvYfA0wvWydw5u0,true
+JIMMY,JIMMY,94XduSfSnyas7jAEFSJSXiCi1xQ4mENWcii1aCvjVuqu,9,https://www.jimmycoinsol.vip/static/media/logo.a7fcc2c98544dcad25a5.png,true
```
Comment (Timoon21) [inline validated-tokens.csv:735]:
```suggestion
JIMMY,JIMMY,94XduSfSnyas7jAEFSJSXiCi1xQ4mENWcii1aCvjVuqu,9,https://arweave.net/5NmSALkMrjr5p33OrjuQlA5a3LRPLv0Ta07SnEWt1fk,true
```

**Review by Timoon21 — CHANGES_REQUESTED** [review]
- [x] On-chain metadata matches the PR
- [ ] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] No duplication on CMC/CoinGecko
- [x] Community validation


Please update the PR to match on-chain metadata

**Review by meijiao — CHANGES_REQUESTED** [review]
pls match onchain meta

**Comment by meijiao** [comment]
Closing this PR -- Please read the review and open a new PR when you are ready.

**Pull request closed without merging**